src/views/network/view_network.cpp

```diff
@@ -81,18 +81,24 @@ namespace big
 
 	void render_session_switcher()
 	{
+		static int selected_region_index     = -1;
+        std::string region_str = (selected_region_index == -1) ? "REGIONS"_T.data() : regions[*g_pointers->m_gta.m_region_code].name;
+
 		ImGui::BeginGroup();
 		components::sub_title("SESSION_SWITCHER"_T);
 		if (ImGui::BeginListBox("###session_switch", get_listbox_dimensions()))
 		{
-			if (ImGui::BeginCombo("##regionswitcher", "REGIONS"_T.data()))
+			if (ImGui::BeginCombo("##regionswitcher", region_str.c_str()))
```

Comment:
This change doesn't make sense to me. There's a drop down box, with no elements near it, besides the join action on the subsequent line. In a perfect UX world, one interaction = one reaction from the software. Since we're not doing that now, we should instead inform the user that selection a new join action will attempt to take them to the new region with a tooltip or something.

Reply:
I may be misunderstanding the ask here. There is no functional change to the behavior; selecting a region from the drop-down doesn't currently do anything besides set the "destination" for the join options on the subsequent lines.

We're just providing feedback to the user that a selection has been made in the menu, without them having to open the menu again to verify.

Reply:
Here is what I'm proposing to guide the user through a typical session join/create workflow.

First-time view:
- The "Select a Region" default drop-down text and tooltip inform the user to take an action
- Until a region is selected, all join/create options are greyed out to prevent behavior that wasn't explicitly intended by the user

![image](https://github.com/YimMenu/YimMenu/assets/13259220/436ce607-b442-4072-af88-db5bd00ce669)

Once a region is selected from the drop-down, it will remain displayed and the previously greyed-out options will be enabled.

![image](https://github.com/YimMenu/YimMenu/assets/13259220/33b17fc1-e11a-4f63-8968-45da57d3bcd5)

I think this is enough to let the user know what needs to be done to execute the feature. Any further thoughts?